python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -126,14 +129,15 @@ def create_channel(
                 cpu_data_typ = SharedMemoryType(
                     buffer_size_bytes=self.buffer_size_bytes
                 )
+                # TODO(sang): We don't support buffers for nccl channel now.
```

Comment:
nit: make the TODO more specific. Is it to support buffers for NCCL channel? Or to consider/evaluate adding the support?